src/components/Preview.css

```diff
@@ -8,7 +8,9 @@
   transform: translateX(-100%);
   transition: 0.2s transform;
   margin-bottom: 0;
-  overflow: scroll;
+  overflow: auto;
+  overflow: -moz-scrollbars-none;
```

Comment:
Remove (browser-specific) unless there's a reason I'm missing for these.

Reply:
Done